.travis.yml

```diff
@@ -8,6 +8,7 @@ matrix:
     - os: osx
       osx_image: xcode8.3
       language: generic  # https://github.com/travis-ci/travis-ci/issues/2312
+      python: 3.6
```

Comment:
This won't work since Travis does not support Python on osx. This is documented one line above.

Reply:
I don't think Homebrew supports installing an arbitrary Python version. It's similar to how apt-get in Ubuntu also doesn't maintain an archive of old Python versions which would let you choose whatever version you'd want to use.

Reply:
So if you currently want to use an old version of Python with a Travis osx instance you're probably better of replacing Homebrew with another Python installation method that is better suited for installing old Python versions.

Reply:
Maybe it is worth pointing out that if the software relies on some version of Python that is very hard to install, then the software itself is also going to be very hard to install because of that dependency.

Reply:
This was an attempt to temporarily fix an issue with the Fabricate script that the project is currently using. I tried updating Fabricate to a later version, but that suffered from different problems.

In any case, I'll probably disable Travis with Kunquat for now, as it has turned out to be too difficult to configure for my needs.